pkg/apis/pipelinesascode/keys/keys.go

```diff
@@ -0,0 +1,51 @@
+/*
+CoCopyright 2021 Red Hat
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package keys
+
+import pipelinesascode "github.com/openshift-pipelines/pipelines-as-code/pkg/apis/pipelinesascode"
+
+const (
+	// Labels and Annotation Keys
+
+	TaskKey            = pipelinesascode.GroupName + "/task"
```

Comment:
can you remove the Key suffix in the var please? 🥺

Reply:
updated.. once CI passes lets merge this.
then I can resolve conflicts for the other one